Inventory Management System/UserInterface.cs

```diff
@@ -10,7 +10,70 @@ internal class UserInterface
     {
         static void Main(string[] args)
         {
+            
 
+            while (true)
+            {
+                Console.WriteLine("Welcome to the Inventory Management System!");
+                DisplayMenu();
+                int choice = GetChoiceFromUser();
+
+                switch (choice)
+                {
+                    case 1:
+                        //  add a product to the inventory
+                        break;
+                    case 2:
+                        // display all products in the inventory
+                        break;
+                    case 3:
+                        // edit an existing product in the inventory
+                        break;
+                    case 4:
+                        //  remove a product from the inventory
+                        break;
+                    case 5:
+                        //search for a product in the inventory
+                        break;
+                    case 6:
+                        Console.Clear();
+                        Console.WriteLine("Exiting the Inventory Management System. Goodbye!");
+                        System.Threading.Thread.Sleep(1000);
+                        return;
+                    
+                        
+                }
+                Console.Clear();
+            }
+        }
+
+        static void DisplayMenu()
```

Comment:
don't use static methods here, please convert the method to a non-static method.